src/couch/src/couch_util.erl

```diff
@@ -38,7 +38,7 @@
 -export([process_dict_get/2, process_dict_get/3]).
 -export([unique_monotonic_integer/0]).
 -export([check_config_blacklist/1]).
--export([check_md5/2]).
+-export([check_digest/2]).
```

Comment:
This one can probably stay renamed as it seems to be generic. It matches on equality or `<<>>`